packages/api/bin/serve.js

```diff
@@ -319,15 +320,23 @@ async function serveApi(user, stackName = localStackName, reseed = true) {
 
     checkEnvVariablesAreSet(requiredEnvVars);
 
-    // create tables if not already created
-    await pRetry(
-      async () => await checkOrCreateTables(stackName),
-      {
-        onFailedAttempt: (error) => console.log(
-          `Failed to Create Tables. Localstack may not be ready, will retry ${error.attemptsLeft} more times.`
-        ),
+    // Create reconciliation report table
```

Comment:
On main we've removed the `checkOrCreateTables` call here which creates all of the Dynamo tables. This _also_ removes it from the v16 release and replaces it with the call to create the reconciliation report table only.